circle.yml

```diff
@@ -0,0 +1,3 @@
+machine:
+  node:
+    version: 5.12.0
```

Comment:
A couple of questions:
- What functionality needs node version `5.12.0`?
- What happens when consumers of this module use node version other than `5.12.0`. Ex: `4.x` or `6.x`?